{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -15,7 +16,7 @@ env:
 install: pip install -U tox
 
 # command to run tests, e.g. python setup.py test
-script: tox
+script: tox -e $TOXENV
```

Comment:
I believe the issue is that your project is using `TOX_ENV` instead of [`TOXENV`](http://tox.readthedocs.org/en/latest/example/general.html#selecting-one-or-more-environments-to-run-tests-against).